config_panel.toml

```diff
@@ -0,0 +1,253 @@
+version = "1.0"
+
+[main]
+name = "Main Settings"
+services = ["matrix-__APP__"]
+
+    [main.welcome]
+    name = "Experience for new Users"
+
+        [main.welcome.enable_registration]
+        ask = "Enable Registration for new users."
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Defaults to 'false'. If 'true', it is highly recommended to use either captcha, email, or token-based verification to avoid SPAM."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [main.welcome.password_enabled]
+        ask = "Enable Password Login?"
+        type = "boolean"
+        yes = true
+        no = false
+        help = "If disabled, Login with Non-YunoHost Users impossible. But it simplies Login process if your Matrix server only has YunoHost SSO Users."
+        bind = "password_config>enabled:/etc/matrix-__APP__/homeserver.yaml"
+        visible = "! enable_registration"
+
+        [main.welcome.registrations_require_3pid]
+        ask = "Registration requires all following 3PID personal identifier."
+        type = "select"
```

Comment:
Wouldn't a type `tags` with choices be better here?

Reply:
Well can you explain exactly what is the difference between `tags` and `select` ?

Reply:
Yes it should be tags, my mistake.
I guess select is a multiple choice, with e.g. checkboxes.

Reply:
@thardev I think select is OK since there is only 4 definite choices. Tags are more adapted to handle a list of arbitrary strings.